src/test/java/uk/gov/companieshouse/ocr/api/image/extracttext/ImageOcrApiControllerTest.java

```diff
@@ -31,7 +31,7 @@
 @Tag(TestType.UNIT)
 @ExtendWith(SpringExtension.class)
 @WebMvcTest(controllers = ImageOcrApiController.class)
-public class ImageOcrApiControllerTest {
+class ImageOcrApiControllerTest {
```

Comment:
add public back. Better explicit. Some languages have opposite defaults. Always good to include accessor level than rely on default. Probably never going to happen but imagine they changed the default?

Reply:
This comment applies to all the other ones...

Reply:
That gives Sonar errors that the lead developers don't want